### ISSUE TITLE
MT Target

### ISSUE THREAD

**Issue by nicedevil-tukui** [Eclipse-/EclipseUI#64]
http://www.abload.de/img/wowscrnshot_022811_192aiwj.jpg

set maintank to true and this happens => maybe you wanna add a unitframe?

oh eclipse, i closed my last issue... did the wrong click with my mouse :( could you reopen it?

**Comment by Eclipse-** [comment]
Yeah I'll get round to finishing the unitframes when I can, most likely for r4.0.4.

No problem about your other issue, it should now be fixed with latest commit.
